horace_core/sqw/file_io/class_helpers/@pix_combine_info/pix_combine_info.m

```diff
@@ -222,8 +246,21 @@
                 obj.(flds{i}) = struc.(flds{i});
             end
         end
+        function pix_range = recalc_pix_range_from_loaders(ldrs)
+            % Recalculate pixels range using list of defined loaders
+            n_files = numel(ldrs);
+            ldr = ldrs{1};
+            pix_range= ldr.get_pix_range();
+            for i=2:n_files
+                ldr = ldrs{i};
+                loc_range = ldr.get_pix_range();
```

Comment:
What is `loc_range`?

Reply:
local range or local variable? The next row makes it entirely clear why it is called this way, as used for calculating `pix_range` (global one)